engine/access/rpc/backend/backend_test.go

```diff
@@ -636,7 +638,7 @@ func (suite *Suite) TestTransactionStatusTransition() {
 		Times(len(fixedENIDs)) // should call each EN once
 
 	// first call - when block under test is greater height than the sealed head, but execution node does not know about Tx
-	result, err := backend.GetTransactionResult(ctx, txID, flow.ZeroID, flow.ZeroID)
+	result, err := backend.GetTransactionResult(ctx, txID, flow.ZeroID, flow.ZeroID, entitiesproto.EventEncodingVersion_JSON_CDC_V0)
```

Comment:
`EventEncodingVersion_JSON_CDC_V0` is used all over the place as default value, is it possible to introduce a variable for it? Something like: `DefaultEventEncodingVersion`?